### ISSUE TITLE
Pre-trained Model Default Parameters

### ISSUE THREAD

**Issue by loretoparisi** [facebookresearch/fastText#256]
I'm using the `wiki.en` and other pre-trained models provided by you (thanks a lot!), but I'm not sure about the training parameters that have been used. [Here](https://github.com/facebookresearch/fastText/blob/master/pretrained-vectors.md) it's stated that the model is based on a `skip-gram` model using default parameters, like 300 features vectors.
Could you please be more specific, about all parameter setup (window, etc.)
How the input text (wikipedia pages) has been normalized?

Thank you.

**Comment by cpuhrsch** [comment]
Hello @loretoparisi,

We now have a script called 'get-wikimedia.sh', that you can use to download and process a recent wikipedia dump of any language. This script applies the preprocessing we used to create the published word vectors.

The parameters we used to build the word vectors are the default skip-gram settings, except with a dimensionality of 300 [as indicated on the top of the list of word vectors](https://github.com/facebookresearch/fastText/blob/master/pretrained-vectors.md) (we now understand that this could be more visible).

Thanks,
Christian
